jgiven-core/src/main/java/com/tngtech/jgiven/impl/Config.java

```diff
@@ -23,69 +28,115 @@
     private static final String JGIVEN_REPORT_TEXT_COLOR = "jgiven.report.text.color";
```

Comment:
Just out of interest: is the public String JGIVEN_REPORT_DIR actually used outside of this file?

Reply:
Yes, [here](https://github.com/TNG/JGiven/blob/b2eaec29df581c1a76ad7c4e45d5352bcaf26f02/jgiven-gradle-plugin/src/main/java/com/tngtech/jgiven/gradle/JGivenPlugin.java#L57).